docs/upgrades/v1.23.3-to-v1.23.4.md

```diff
@@ -0,0 +1,265 @@
+
+This guide describes the steps to follow to upgrade the Kubernetes Fury Distribution (KFD) from v1.23.3 to 1.23.4.
+
+If you are running a custom set of modules, or different versions than the ones included with each release of KFD, please refer to each module's release notes.
+
+Notice that the guide will not cover changes related to the cloud provider, ingresses or pod placement changes. Only changes related to KFD and its modules.
+
+> ⛔️ **IMPORTANT**
+> we strongly recommend reading the whole guide before starting the upgrade process to identify possible blockers.
+<!-- separator -->
+> ⚠️ **WARNING**
+> the upgrade process involves downtime of some components.
+
+## Upgrade procedure
+
+As a high-level overview, the upgrade procedure consists on:
+
+1. Upgrading KFD (all the core modules).
+2. Upgrading the Kubernetes cluster itself.
+
+### 1. Upgrade KFD
+
+The suggested approach to upgrade the distribution is to do it one module at a time, to reduce the risk of errors and to make the process more manageable.
+
+#### Networking module upgrade
+
+To upgrade the Networking module to the new version, update the version on the `Furyfile.yml` file to the new version:
+
+```yaml
+versions:
+  networking: v1.12.1
+...
+```
+
+Then, download the new modules with `furyctl` with the following command:
+
+```bash
+furyctl vendor -H
+```
+
+Apply your Kustomize project that uses Networking module packages as bases with:
+
+> ⛔️ **IMPORTANT** you may want to limit the scope of the command to only the networking module, otherwise, the first time you apply with `--server-side` other pods may also be restarted.
+>
+> The same applies to the rest of the modules, we will not include this warning in every step for simplicity.
+
+```bash
+kustomize build <your-project-path/monitoring-base> | kubectl apply -f - --server-side --force-conflicts
+```
+
+Wait until all Calico pods are restarted and running. You can check Calico's Grafana dashboard "General / Felix Dashboard (Calico)" and the "Networking / *" dashboards to make sure everything is working as expected.
+
+#### Monitoring module upgrade
+
+Monitoring module has not been upgraded.
+
+#### Logging module upgrade
+
+> ⚠️ **WARNING** downtime of the Logging stack is expected during this process.
+
+To upgrade the Logging module to the new version, update the version on the `Furyfile.yml` file to the new version:
+
+```yaml
+versions:
+...
+  logging: v3.1.3
+...
+```
+
+Then, download the new modules with `furyctl` with the following command:
+
+```bash
+furyctl vendor -H
+```
+
+Since this upgrade changes the major version, there are some manual steps involving breaking changes that you need to do before applying the project:
+
+Remove the old `minio` stack:
+
+```bash
+kubectl delete sts minio -n logging
+```
+
+Apply your Kustomize project that uses Logging module packages as bases with:
+
+```bash
+kustomize build <your-project-path> | kubectl apply -f - --server-side --force-conflicts
+```
+
```

Comment:
should we add some instructions for Loki? I guess not but maybe I'm missing something.

Reply:
I don't think so, to use loki the instructions are on the logging module readme